core/src/main/resources/reference.conf

```diff
@@ -759,6 +759,16 @@ datastax-java-driver {
       # Overridable in a profile: yes
       consistency = ONE
     }
+
+    # Whether logging of server warnings generated during query execution should be disabled by the
+    # driver. All server generated warnings will be available programmatically via the ExecutionInfo
+    # object on the executed statement's ResultSet. If set to "true", this will prevent the driver
+    # from logging these warnings.
```

Comment:
We should also mention that it will reuse the formatting options in `advanced.request-tracker`.